environment.yml

```
@@ -0,0 +1,7 @@
+# configuration for binder
+
```

```suggestion
channels:
  - conda-forge
  - defaults
```
I think what might be happening is that we need to specify to look in `conda-forge`, since that is what PlasmaPy is available through.